application/config.py

```diff
@@ -4,4 +4,4 @@
 
 
 DEED_API_BASE_HOST = os.getenv('DEED_API_ADDRESS',
-                               'http://deed-api.dev.service.gov.uk')
+                               'http://10.10.10.10:9020')
```

Comment:
Why was this changed to a localhost address? In production we should have the default pointing to whatever the LIVE address is.